cosmos/providers/dbt/core/operators/kubernetes.py

```diff
@@ -141,9 +141,13 @@ class DbtTestKubernetesOperator(DbtKubernetesBaseOperator):
 
     ui_color = "#8194E0"
 
-    def __init__(self, **kwargs) -> None:
+    def __init__(
+        self, on_warning_callback: Optional[Callable] = None, **kwargs
+    ) -> None:
         super().__init__(**kwargs)
         self.base_cmd = "test"
+        # as of now, on_warning_callback in kubernetes executor does nothing
+        self.on_warning_callback = on_warning_callback
```

Comment:
if it doesn't do anything, can we remove it as an arg until it _is_ supported?

Reply:
same as in the docker case